src/create-model.tsx

```diff
@@ -18,11 +18,14 @@ export function createModel<T, P>(hook: ModelHook<T, P>, hookArg?: P) {
   );
   const useModel: UseModel<T> = depsFn => {
     const [state, setState] = useState<T | undefined>(() =>
-      container ? (container.data as T) : undefined
+      container ? container.data : undefined
     );
     const depsFnRef = useRef(depsFn);
     depsFnRef.current = depsFn;
-    const depsRef = useRef<unknown[]>([]);
+    const depsRef = useRef<unknown[]>(
+      depsFnRef.current?.(container.data) || []
```

Comment:
这里是不是写错了

```suggestion
      depsFnRef.current ? container.data : []
```

`depsFn` 上为啥会有 `container` 属性

Reply:
没写错的～ 这里是初始化执行依赖函数 `depsFnRef.current?.(container.data)` 是depsFnRef.current && depsFnRef.current(container.data)的简写

Reply:
噢噢我看错了